src/d3a/models/strategy/pv.py

```diff
@@ -245,8 +250,13 @@ def _delete_past_state(self):
         for k in to_delete:
             self.state.available_energy_kWh.pop(k, None)
             self.energy_production_forecast_kWh.pop(k, None)
+            del self.offer_update.initial_rate[k]
```

Comment:
```suggestion
            del self.offer_update.initial_rate[market_time_slot]
```
...and following the same convention inside `_delete_past_state `

Reply:
Sorry, i copied this from the other `_delete_past_state` and there a `k` . 
If you like, will refactor this.

Reply:
Always better to have self-explanatory as well as precise variable name than one letter variable.
..and I didn't mean to point at you, just saw it while reviewing and see a potential enhancement.